composer.json

```diff
@@ -44,7 +44,9 @@
         "phpstan/phpstan-strict-rules": "^0.12.2",
         "phpunit/phpunit": "9.5.0",
         "psalm/plugin-phpunit": "0.13.0",
+        "psr/cache": "^1.0 || ^2.0 || ^3.0",
         "squizlabs/php_codesniffer": "3.6.0",
+        "symfony/cache": "^5.2",
```

Comment:
Why is a dependency on a specific cache implementation needed? If it's for testing, should it be made a development dependency?

Reply:
@morozov it is a development dependency.

Reply:
I can remove `psr/cache` indeed. But we need an implementation to test against, so either `symfony/cache` or any other PSR-6 implementation you want should remain in `require-dev`.

Reply:
The `psr/cache` dependency is gone now.

Reply:
What I meant to say is that there should probably be a dependency on `psr/cache`, but in `require`? Or is the reasoning that the dependency is optional. Since it's mentioned in `src`, it should appear in one of these sections IMO, sorry for the misunderstanding.

Reply:
No worries, I've added it to the `require` section now.